src/frontends/pytorch/src/op/aminmax.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/op/reduce_max.hpp"
+#include "openvino/op/reduce_min.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_aminmax(const NodeContext& context) {
+    num_inputs_check(context, 1, 4);  // Expect between 1 and 4 inputs
+                                      // (input tensor, dim = none, keepdim = false, out = none)
+
+    auto input = context.get_input(0);
+
+    // check if dim is provided, if not, get the range of axes to compute min and max
+    auto dim = !context.input_is_none(1) ? context.get_input(1) : get_axes_range(context, 0);
+
+    // check if keepdim is provided, if not, set it to false like PyTorch
+    bool keep_dims = !context.input_is_none(2) ? context.const_input<bool>(2) : false;
+
+    auto amin = context.mark_node(std::make_shared<v1::ReduceMin>(input, dim, keep_dims));
+    auto amax = context.mark_node(std::make_shared<v1::ReduceMax>(input, dim, keep_dims));
+
+    if (!context.input_is_none(3)) {
+        auto concat = context.mark_node(std::make_shared<v0::Concat>(OutputVector{amin, amax}, 0));
+        context.mutate_input(3, concat);
```

Comment:
`out` is a tuple in torch, so this will not work. You would need to access each element of the tuple and update it using low level functions of context and decoder like `add_tensor_to_context`. That may be tricky to do, so I would suggest to implement this conversion only for case when `out` is not provided. Just validate that out is None

Reply:
> `out` is a tuple in torch, so this will not work. You would need to access each element of the tuple and update it using low level functions of context and decoder like `add_tensor_to_context`. That may be tricky to do, so I would suggest to implement this conversion only for case when `out` is not provided. Just validate that out is None


Yes, I knew I had to return a tuple since PyTorch was returning one, but since a tuple is immutable it was causing problems in my mind... 
Anyway, I added a check and an assert in case out is used as a parameter.